### DISCUSSION TITLE
"blog offline" on password protected blog

### DISCUSSION THREAD

**Discussion by nuvema572** [TumblThreeApp/TumblThree#311] [Q&A (Question & Answer)]
hi,
upon trying to download a password protected blog, it is consistently showing as "offline". the password is properly entered and i am logged into tumblr via the authentication thing. despite being password protected it's not a login-required/dashboard only blog. 
i can download other blogs that aren't pass-protected just fine. i have tried re-adding and deleting it from the list of blogs as well, and "check blog online status at startup" is on. i am using version 2.5.5.286 and my browser is firefox.

this is my blog that i own so i don't understand why there should be an issue unless i'm doing something incorrectly :( thankyou for your help

**Comment by thomas694** [comment]
Hi,
I don't think you are doing something incorrectly. After a short test it seems as if the download of password protected blogs is broken, maybe already since a while ago as not many seem to use this feature.

**Comment by nuvema572** [comment]
all is well, everything seemed to fix itself somehow. i'm not even sure what i did to fix it, i just readded it in the queue a couple more times? just takes a bit to get moving i guess
